transitions/registration.js

```diff
@@ -272,5 +252,43 @@ module.exports = {
                 callback();
             }
         });
+    },
+    addPatient: function(options, callback) {
+        var doc = options.doc,
+            db = options.db,
+            audit = options.audit,
+            id = doc.patient_id,
+            patientNameField = (options.params && options.params.length && options.params[0]) || 'patient_name';
+
+        utils.getRegistrations({
+            db: db,
+            id: id
+        }, function(err, registrations) {
+            if (err) {
+                return callback(err);
+            }
+            if (registrations.length) {
+                // patient already registered, no action required
+                return callback();
+            }
+            db.medic.view('medic-client', 'people_by_phone', {
+                key: [ doc.from ],
+                include_docs: true
+            }, function(err, result) {
+                if (err) {
+                    return callback(err);
+                }
+                var contact = _.result(_.first(result.rows), 'doc');
+                // create a new patient with this patient_id
+                var patient = {
+                    name: doc.fields[patientNameField],
+                    parent: contact && contact.parent,
```

Comment:
Hmmmwokay I guess. (you gave it some thought, it's not an omission, which is the main thing I'm checking!)

Where does the patient show up if it's an orphan? Does the new People tab design still allow to see them? (there are less things displayed in the RHS now) Does it also depend on permissions? 
Not necessarily blocker for here, but if it's not clear please file an separate issue to figure out the UX flow of orphan patients, as part of People tab redesign.